source/includes/important-delayed-replica-set-members.rst

```diff
@@ -0,0 +1,27 @@
+.. important::
+
+   If your replica set contains :doc:`delayed members
+   </core/replica-set-delayed-member>` ensure that the delayed
+   members are hidden and non-voting. 
+
+   Hiding delayed replica set members prevents applications from seeing 
+   and querying delayed data without a direct connection to that member.
+   Making delayed replica set members non-voting means they will not 
+   count towards acknowledging write operations with read concern 
+   :readconcern:`"majority"`.
+
+   If you do not hide delayed members and one or more nodes 
+   becomes unavailable, the replica set has to wait for the delayed 
+   member and the commit point lags. A lagged commit point can lead to
+   performance issues.
+
+   For example:
+
+   - Primary-Secondary-Delayed replica set config where delayed 
+     secondary is voting with 10 mins delay.
```

Comment:
This bullet is a sentence fragment which reads a little awkwardly. See suggestion in the comment above.

Reply:
Updated.